src/nvim/keymap.h

```diff
@@ -209,9 +209,9 @@ enum key_extra {
   , KE_S_XF3 = 73
   , KE_S_XF4 = 74
 
-  // NOTE: The scroll wheel events are inverted: i.e. UP is the same as
-  // moving the actual scroll wheel down, LEFT is the same as moving the
-  // scroll wheel right.
+               // NOTE: The scroll wheel events are inverted: i.e. UP is the same as
+               // moving the actual scroll wheel down, LEFT is the same as moving the
+               // scroll wheel right.
```

Comment:
kinda weird but not super important, doesnt seem to be common.

Reply:
That's _probably_  due to the Haskell-style enum...

Reply:
Confirmed! Moving the commas the end fixes the comments.